lib/graphql/schema.ts

```diff
@@ -1055,6 +1054,14 @@ export type TransactionsConnectionEdge = {
   cursor: Scalars['String'],
 };
 
+export type TransactionSplit = {
+   __typename?: 'TransactionSplit',
+  id: Scalars['ID'],
```

Comment:
should this be optional? `id?: Scalars['ID']` since when creating a transaction split there won't be an ID created yet?